src/pwa-buildpack/project-setup/create-simple-peregrine-app/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Create simple peregrine app
+---
+
+In the [previous topic], you created configuration files for your theme project.
+In this topic, you will create a simple app that follows the [Peregrine] pattern.
+
+## Create app.js
+
+1. Inside `src`, create a directory called `components`.
+1. Inside the `components` directory, create an `app.js` file with the following content:
+
+   ``` jsx 
+   import React from 'react';
+
+    export default class App extends React.Component {
+        render() {
+            return (
+                <h1>
+                    Hello, Studio!
+                </h1>
+            );
+        }
+    }
+
+   ```
+
+   This code defines a React component that returns "Hello, Studio!" inside `h1` tags.
+
+## Create index.js
+
+Inside `src` create an `index.js` file with the following content:
+
+``` javascript
+import Peregrine from '@magento/peregrine';
+import App from './components/app';
+
+const app = new Peregrine();
+const container = document.getElementById('root');
+
+app.component = App;
+app.mount(container);
+
+if ('serviceWorker' in navigator) {
+    window.addEventListener('load', () => {
+        navigator.serviceWorker
+            .register(process.env.SERVICE_WORKER_FILE_NAME);
+    });
+}
+
+export default app;
+```
+
+This imports the previously-defined `app` component and incorporates it into Peregrine before adding it to an HTML `root` element. 
+
+## Create RootComponents directory
+
+Inside `src`, create a `RootComponents` directory.
+This directory will contain code for your root components.
+For now, leave this directory empty.
+
+## Congratulations!
+
+You have created a Peregrine app skeleton!
+
+If you followed the [Project setup tutorial] from the beginning, you have also successfully set up your development environment!
```

Comment:
This isn't true! Whoops! You also need to login to your Magento instance, go to `Design -> Themes` and check to see if your PWA theme is successfully installed and linked. Then you have to go to `Design -> Configuration` and set your site to use that theme. Only then can you launch a development environment! (We'll want to automate this in the future, as we will many things.)

Reply:
I think it makes sense to add the check and install instructions in the Link Project step